plugin/trino-bigquery/pom.xml

```diff
@@ -22,7 +22,7 @@
             <dependency>
                 <groupId>com.google.cloud</groupId>
                 <artifactId>libraries-bom</artifactId>
-                <version>26.14.0</version>
+                <version>26.18.0</version>
```

Comment:
> Update apache arrow to 12.0.1

The commit title and the actual change don't match.

Reply:
Yeah. I started with the arrow but in order to update it i needed to sync other dependencies like bigquery or protobuf. Logically it’s a single unit of change. I’ll improve commit message

Reply:
I squashed two commits together and fixed H2 test assertion which was the only failure in the tests.